packages/react/src/components/Provider/index.tsx

```diff
@@ -80,6 +81,8 @@ export const CunninghamProvider = ({
 
         return message;
       },
+      currentLocale:
+        (locales[currentLocale] && currentLocale) || DEFAULT_LOCALE,
```

Comment:
This logic could be factorized with the `locale` computation. WDYT @jbpenrath ?

Reply:
Would you want me to add a test for this new feature ?

Reply:
Yup please